### DISCUSSION TITLE
List of exception numbers?  What is exception 2

### DISCUSSION THREAD

**Discussion by dave-vsdevs** [arendst/Tasmota#13352] [Q&A] [answered]
Is there a list of exceptions anyplace?   I dug thru the code for a bit and couldn't find any

{"Info3":{"RestartReason":{"Exception":**2**,"Reason":"Exception","EPC":["3ffef408","00000000","00000000"],"EXCVADDR":"3ffef408","DEPC":"00000000","CallChain":["402244f6","40100cf2","402164c9","4022f062","4022f084","401007ec","4023bb08","40101985"]}}}

**Comment by sfromis** [comment, marked as answer]
See https://links2004.github.io/Arduino/dc/deb/md_esp8266_doc_exception_causes.html

2 is InstructionFetchErrorCause - Processor internal physical address or data error during instruction fetch
